src/librustc/ty/sty.rs

```diff
@@ -254,6 +254,41 @@ pub enum TyKind<'tcx> {
     Error,
 }
 
+impl<'tcx> TyKind<'tcx> {
+    pub fn article_and_description(&self) -> (&'static str, &'static str) {
```

Comment:
This doesn't seem to be used any more, and is somewhat duplicating `sort_string`